logger/module.go

```diff
@@ -60,17 +61,24 @@ func (*loggerModule) Create(runtimeMgr modules.RuntimeMgr) (modules.Module, erro
 	m.InitLogger()
 
 	// Mapping config string value to the proto enum
-	if pocketLogLevel, ok := LogLevel_value[`LogLevel_LOG_LEVEL_`+strings.ToUpper(m.config.GetLevel())]; ok {
+	if pocketLogLevel, ok := LogLevel_value[`LOG_LEVEL_`+strings.ToUpper(m.config.GetLevel())]; ok {
 		zerolog.SetGlobalLevel(pocketLogLevelToZeroLog[LogLevel(pocketLogLevel)])
 	} else {
 		zerolog.SetGlobalLevel(zerolog.NoLevel)
 	}
 
 	if pocketLogFormatToEnum[m.config.GetFormat()] == LogFormat_LOG_FORMAT_PRETTY {
-		mainLogger = mainLogger.Output(zerolog.ConsoleWriter{Out: os.Stderr})
+		logStructure := zerolog.ConsoleWriter{Out: os.Stdout}
+		logStructure.FormatLevel = func(i interface{}) string {
+			return strings.ToUpper(fmt.Sprintf("[%s]", i))
+		}
+
+		mainLogger = mainLogger.Output(logStructure)
 		mainLogger.Info().Msg("using pretty log format")
 	}
 
+	Global = m.CreateLoggerForModule("global")
```

Comment:
So if global doesn't exist until we call `Create()` on `LoggerModule`, wouldn't that make it impossible to use global logger before `LoggerModule` is initiated? I suspect there might be some log lines we want to log before we init the modules.